modules/install-secured-cluster-operator.adoc

```diff
@@ -26,16 +26,7 @@ To install Collector on systems that have Unified Extensible Firmware Interface
 
 .Procedure
 . On the {ocp} web console, navigate to the *Operators* -> *Installed Operators* page.
-. Select the {product-title} Operator from the list of installed operators.
-.. Select the *Installation mode* of *All namespaces on the cluster (default)*. 
-.. Under *Installed Namespace*, select the default project of `rhacs-operator`. If this namespace does not exist, the installation program creates it.
-+
-[WARNING]
-====
-You must install {product-title} `SecuredCluster` resource in its own project and not the default `openshift-operators` project.
-====
-.. Click *Install*. 
-. After the Operator is installed, click *View Operator*. 
+. Click the {product-title-short} Operator.
 . Select *Secured Cluster* from the central navigation menu in the *Operator details* page.
```

Comment:
This is out of the scope of your PR but I wonder if it would be worth editing these bullets with "Click" since select is basically the same thing?